### PR TITLE
give the app toolbox tutorials some writer TLC

### PR DESCRIPTION
**Pull request by kaycebasques**
Plz R @arthurevans

### CODE REVIEW

File: app/1.0/start/toolbox/add-elements.md [hunk 1]
```diff
@@ -1,84 +1,71 @@
 ---
-title: Step 3. Add some elements
+title: Step 3. Add an element
 subtitle: "Build an app with App Toolbox"
 ---
 
 <!-- toc -->
 
-Now that you've added a new view to your application, you can start building
-out the details of that view.
```
Comment (arthurevans) [inline app/1.0/start/toolbox/add-elements.md]: This deleted sentence actually introduces the step—so some variant of this should probably stay here. (And we should keep the term "view" which is used throughout the tutorial).

File: app/1.0/start/toolbox/add-elements.md [hunk 2]
```diff
@@ -1,84 +1,71 @@
...
+One of the appeals of the web component approach to web development 
+is the ecosystem. Rather than building everything in your app from scratch,
+you can find and use third-party elements that suit your needs. Two examples
```
Comment (arthurevans) [inline app/1.0/start/toolbox/add-elements.md]: "third-party" isn't correct here (that suggests a component built by someone who isn't us and isn't the developer we're talking to). Could use 'ready-made' or 'off-the-shelf'.

**Comment by addyosmani** [comment]
Friendly ping @kaycebasques on the review from @arthurevans above 🍰

**Comment by kaycebasques** [comment]
I updated the PR per Arthur's feedback. Had trouble with merge conflicts though, hence the googlebot...

**Comment by arthurevans** [comment]
Yeah, that commit from dandv should not be showing up in your PR. Not sure what's going on there. You may want to create a clean branch off of master and cherry-pick your commits into it.

Strangely enough, your original commit isn't showing up here anymore... Did you squash it into the new one?

**Comment by arthurevans** [comment]
Closing in favor of #1767.

**Pull request closed without merging**